src/inference/src/dev/core_impl_ie.cpp

```diff
@@ -241,43 +183,12 @@ InferenceEngine::QueryNetworkResult ov::CoreImpl::QueryNetwork(const InferenceEn
         return ret;
     }
     auto res = query_model(network.getFunction(), deviceName, any_copy(config));
-    if (!network.getFunction() || res.empty()) {
+    if (res.empty()) {
```

Comment:
why it is an error? It's valid situation if plugin does not support ALL operations from model

Reply:
I didn't change the legacy logic :)

Reply:
But answer that I changed the logic